_inc/client/plans/style.scss

```diff
@@ -257,3 +263,8 @@ $plan-features-sidebar-width: 272px;
 		margin: 0 1px;
 	}
 }
+
+.single-product__landing-page {
+  text-align: center;
```

Comment:
You'll want to use tabs instead of spaces here, or the next person editing this file may transform all those on save.

Reply:
I went ahead and applied a Prettier transformation on this file.